package.json

```diff
@@ -49,8 +49,8 @@
   },
   "dependencies": {
     "algoliasearch": "^3.8.0",
-    "algoliasearch-helper": "^2.4.0",
-    "classnames": "^2.1.5",
+    "algoliasearch-helper": "^2.5.0",
+    "classnames": "^2.1.3",
```

Comment:
Yes but here it's the classname dependency from package.json that was updated, I do not get why you would need to update the classname dependency. It's not used by the helper. You can just ignore the change by editing the commit, should be fine.